docs/source/settings.rst

```diff
@@ -31,6 +31,26 @@ Whether to enable Spidermon.
 SPIDERMON_EXPRESSIONS_MONITOR_CLASS
 -----------------------------------
 
+
+Default: ``spidermon.python.monitors.ExpressionMonitor``
+
+A fully qualified path for a Python Class that should inherit from 
+``spidermon.python.monitors.ExpressionMonitor``.
+
+This class will be used to generate on-the-fly monitors for the folowing settings:
+
+    * :ref:`SPIDERMON_SPIDER_CLOSE_EXPRESSION_MONITORS`
+    * :ref:`SPIDERMON_SPIDER_OPEN_EXPRESSION_MONITORS`
+    * :ref:`SPIDERMON_ENGINE_STOP_EXPRESSION_MONITORS`
+
+Please refer to :ref:`expression-monitors` to better understand what is
+a Expression Monitor.
+
+.. note::
+    Your probably wont change this setting unless you have an advanced use case and
+    needs to change how the context data is build or how the on-the-fly ``MonitorSuite``
+    are generated. Otherwise the default should be enough
```

Comment:
I think this note is really helpful, although in the future, when we extend the documentation about expression monitors to cover when and how to change this setting, I would remove this note as well.